### PR TITLE
adobe-air-sdk, adobe-air-sdk-flex: update to 17.0.0.172

### PR DESCRIPTION
**Pull request by dy-dx**

### CODE REVIEW

File: adobe-air-sdk-flex.rb [hunk 1]
```diff
@@ -20,6 +20,6 @@ def caveats; <<-EOS.undent
   end
 
   test do
-    system "#{bin}/fontswf", "-h"
+    system "#{bin}/adt", "-version"
```
Comment (dy-dx) [inline adobe-air-sdk-flex.rb]: Only the adl & adt binaries are available in this package

**Comment by dy-dx** [comment]
The failing audits in the build report can be ignored, according to discussion here: https://github.com/Homebrew/homebrew/pull/34122

**Comment by sugamasao** [comment]
hi.
This p-r will not be merged?

I tried the master, but  error occurred :sweat_drops: 
This Download `bin/adt` version is `17.0.0.172`.
```
$ brew install adobe-air-sdk-flex
==> Installing adobe-air-sdk-flex from homebrew/homebrew-binary
==> Downloading http://airdownload.adobe.com/air/mac/download/17.0/AdobeAIRSDK.tbz2
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: b13fcea757349692d82c14926d9afb1477387af0f3808f8ef7dbb7c373d3eb38
Actual: c71708fcfb82a3d3c6a78a3932e498b059ecccb72aa8bba123594d9f50622f3f
Archive: /Library/Caches/Homebrew/adobe-air-sdk-flex-17.0.0.144.tbz2
To retry an incomplete download, remove the file above.
```

**Comment by vpmedia** [comment]
Please review/vote on this request:
https://github.com/Homebrew/homebrew-binary/pull/228

**Comment by dunn** [comment]
@BrewTestBot test this please

**Comment by dy-dx** [comment]
Ooh this is outdated. Air v18 is out now.

**Pull request closed without merging**